test/check-terminal

```diff
@@ -26,6 +26,18 @@ class TestTerminal(MachineCase):
         m = self.machine
         b.default_user = "admin"
 
+        # XXX - figure out why locale isn't set up properly by default
+        if "debian" in m.image:
+            m.execute("echo 'en_US.UTF-8' >>/etc/locale.gen; locale-gen")
+            m.execute("echo 'export LANG=en_US.UTF-8' >>/home/admin/.bashrc")
```

Comment:
It turns out that the images were already correct, but the PAM file I am using for tests did not setup the environment correctly.